### DISCUSSION TITLE
Welcome to Olauncher Discussions!

### DISCUSSION THREAD

**Discussion by tanujnotes** [tanujnotes/Olauncher#53] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maifeeulasad** [comment]
Question - Do you accept pull request or have plan to accept contribution from others ?

*Loved your work*

**Reply by tanujnotes** [reply]
Thank you. I'll accept bug fixes but anything else, probably not. If someone wants to contribute, add features etc, they can discuss with me.

**Comment by LuisSousaRego** [comment]
Hey, I've been using the Olauncher for a few days and I really like it. Nonetheless I have a couple of suggestions:
- option to see apps icons on the search list. Some apps I use have the same name and without an icon I need to guess which one I want to open.
- option to change font. I see you want to implement text size and color, also adding the possibility to choose from a small list of font families like serifed, monospace and sans serif would be great.

**Reply by tanujnotes** [reply]
Thank you for the feedback. Olauncher will most probably never have app icons. I'll what I can do about the rest.

**Comment by Guestuser786** [comment]
I really like your launcher. But I suggest a feature **show hidden apps**. Thanks.

**Comment by icedragal** [comment]
Hi! I've been using this launcher for a week and I'm very happy with it. Thank you for the work done. Can you tell me if I can see all the background images somewhere? I would like to put install them on the lock screen so that it matches the background of the launcher.

**Reply by tanujnotes** [reply]
Hi, if you tap on 'Daily new wallpaper' text in Olauncher settings, you can download the wallpaper.

For complete feature list, please visit the About page in settings. Thank you.
